shiftorg/skills
# Resume form

This PR introduces a resume-submission form! Still kind of rough, but this is a proof of concept that we can take a submission, send the text to Flask, do something with that text on the Python side, and then get the result of that processing back up to the front end.

To test:

```
docker-compose build
docker-compose up -d
```

then navigate to `localhost:5090` in Google Chrome . Try pasting in a resume or job description (or just typing!) and then hit "submit". You should see the tokenized response shown below it!
